src/CookBook/CookBook.Mobile/ViewModels/Recipe/RecipeDetailViewModel.cs

```diff
@@ -1,57 +1,48 @@
-﻿using CommunityToolkit.Mvvm.Input;
+﻿using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
 using CookBook.Common.Models;
 using CookBook.Mobile.Api;
-using PropertyChanged;
 
 namespace CookBook.Mobile.ViewModels;
 
 [QueryProperty(nameof(Id), "id")]
-public partial class RecipeDetailViewModel : ViewModelBase
-{
```

Comment:
Good brace formatting, but is not consistent with the rest of the project ;)

Reply:
Yeah, new Visual Studio Preview started doing this as default formatting. I'll fix it manually.